scripts/hash-exec-copy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$1" =~ https://raw.githubusercontent.com/makerdao/community/* ]] || { echo "Please provide the exec copy link to hash (e.g. url=<https://raw.githubusercontent.com/makerdao/community/...>)"; exit 1; }
+
+for ARGUMENT in "$@"
+do
+    KEY=$(echo "$ARGUMENT" | cut -f1 -d=)
+    VALUE=$(echo "$ARGUMENT" | cut -f2 -d=)
+
+    case "$KEY" in
+            url)      URL="$VALUE" ;;
+            *)        URL="$KEY" 
+    esac
+done
+
+if ! [[ -x "$(command -v wget)" ]]; then
+    cast keccak -- "$(curl "$URL" -o - 2>/dev/null)"
```

Comment:
I think we should also make sure `curl` is available, otherwise to throw an error saying that `curl` or `wget` should be installed.

Reply:
in the current setup, `curl` is a fallback for `wget` in case it's not installed, mostly for mac users where I think `wget` is not installed by default.

Reply:
it should be addressed in 89705da

Reply:
Looks good now!